Source/Blazorise/Components/MessageAlert/MessageAlert.razor.cs

```diff
@@ -229,57 +243,85 @@ protected virtual Color CancelButtonColor
         protected virtual string CancelButtonClass
             => Options?.CancelButtonClass;
 
+        /// <summary>
+        /// Gets the OK button padding.
+        /// </summary>
+        protected virtual IFluentSpacing OkButtonPadding
+            => Options?.OkButtonPadding ?? Blazorise.Padding.Is2.OnX;
+
+        /// <summary>
+        /// Gets the cancel button padding.
+        /// </summary>
+        protected virtual IFluentSpacing CancelButtonPadding
+            => Options?.CancelButtonPadding ?? Blazorise.Padding.Is2.OnX;
+
+        /// <summary>
+        /// Gets the confirm button padding.
+        /// </summary>
+        protected virtual IFluentSpacing ConfirmButtonPadding
+            => Options?.ConfirmButtonPadding ?? Blazorise.Padding.Is2.OnX;
+
         /// <summary>
         /// Gets or sets the <see cref="IMessageService"/> to which this dialog is responding.
         /// </summary>
-        [Inject] protected IMessageService MessageService { get; set; }
+        [Inject]
```

Comment:
If get set are on the same line we prefer to also have attributes on the same line.